packages/block-library/src/spacer/edit.js

```diff
@@ -6,24 +6,28 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
-import { useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import { InspectorControls } from '@wordpress/block-editor';
-import { BaseControl, PanelBody, ResizableBox } from '@wordpress/components';
+import { PanelBody, ResizableBox, RangeControl } from '@wordpress/components';
 import { compose, withInstanceId } from '@wordpress/compose';
 import { withDispatch } from '@wordpress/data';
 
+const minSpacerHeight = 20;
+const maxSpacerHeight = 500;
```

Comment:
The WordPress JavaScript Coding Standards [recommend using CAPS_WITH_UNDERSCORES for constants like these](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#constants).

Reply:
Updated the code to follow the recommendations.